src/main/java/com/github/onsdigital/babbage/api/endpoint/search/SearchData.java

```diff
@@ -31,8 +37,15 @@
     @GET
     public void get(HttpServletRequest request, HttpServletResponse response) throws Exception {
         String searchTerm = extractSearchTerm(request);
-        search(request, getClass().getSimpleName(), searchTerm, queries(request, searchTerm), false)
-                .apply(request, response);
+        if (searchTerm != null) {
+            String timeSeriesRedirect = getRedirectWhenTimeSeries(request, searchTerm);
+            if (timeSeriesRedirect != null) {
+                new BabbageRedirectResponse(timeSeriesRedirect, appConfig().babbage().getSearchResponseCacheTime())
+                        .apply(request, response);
+            }
+        }
+        Map<String, SearchResult> results = search(searchTerm, queries(request, searchTerm), false);
+        SearchRendering.buildResponse(request, getClass().getSimpleName(), results).apply(request, response);
```

Comment:
Same question here. `BabbageRedirectResponse.apply` writes the data to the underlying HTTP response body - I'm not sure what happens in the `SearchRendering.buildResponse` method - are we writing to the same response again?

Reply:
As discussed - if possible can we unit test this?